### PR TITLE
Log the version number on agent start.

### PR DESCRIPTION
**Pull request by igorpeshansky**

### CODE REVIEW

File: templates/etc/init.d/deb/td-agent [hunk 1]
```diff
@@ -202,7 +203,7 @@ do_configtest() {
 RETVAL=0
 case "$1" in
 "start" )
-  echo -n "Starting ${TD_AGENT_NAME}: "
+  echo -n "Starting ${TD_AGENT_VERSION}: "
```
Comment (dhrupadb) [inline templates/etc/init.d/deb/td-agent]: Wouldn't it make sense to keep the Agent Name as well? Say change it to 
`echo -n "Starting ${TD_AGENT_NAME}-${TD_AGENT_VERSION}:" `

Comment (igorpeshansky) [inline templates/etc/init.d/deb/td-agent]: I was being lazy in my variable naming. `TD_AGENT_VERSION` actually includes both name and version.

**Comment by igorpeshansky** [comment]
Ping @dhrupadb and @qingling128. Is this ok to go in?

File: templates/etc/init.d/deb/td-agent (hunk 1 above)
Comment (dhrupadb) [inline templates/etc/init.d/deb/td-agent]: I think it would be a nice to have for anyone just browsing this code later.

**Comment by dhrupadb** [comment]
Sorry for the delay. Pending the renaming suggestion LGTM.

**Comment by qingling128** [comment]
LGTM.

File: templates/etc/init.d/deb/td-agent (hunk 1 above)
Comment (igorpeshansky) [inline templates/etc/init.d/deb/td-agent]: Done.

**Comment by igorpeshansky** [comment]
@dhrupadb PTAL.

**Comment by dhrupadb** [comment]
LGTM

**Pull request merged**